tests/fixtures/expected_results_sentinel2.py

```diff
@@ -0,0 +1,10 @@
+
+
+filter = [{'sat': 'S2A', 'path': 'tiles/22/K/HV/2017/1/12/0/', 'utm_zone': '22', 'latitude_band': 'K', 'grid_square': 'HV', 'num': '0', 'acquisition_date': '20170112', 'browseURL': 'https://sentinel-s2-l1c.s3.amazonaws.com/tiles/22/K/HV/2017/1/12/0/preview.jpg', 'scene_id': 'S2A_tile_20170112_22KHV_0'}, {'sat': 'S2A', 'path': 'tiles/22/K/HV/2
```

Comment:
@pablotcarreira is this used somewhere ?

Reply:
It was the remnant of an ancient methodology. I removed it.